server/enterprise/message_export/actiance_export/actiance_export.go

```diff
@@ -75,12 +75,32 @@ type LeaveExport struct {
 
 // The Message element indicates the message sent by a user
 type PostExport struct {
-	XMLName      xml.Name `xml:"Message"`
-	UserEmail    string   `xml:"LoginName"`    // the email of the person that sent the post
-	UserType     string   `xml:"UserType"`     // the type of the person that sent the post
-	PostTime     int64    `xml:"DateTimeUTC"`  // utc timestamp (seconds), time at which the user sent the post. Example: 1366611728
```

Comment:
Is removing this going to cause issues in terms of compatibility? Asking as I am not sure how these exports are managed/parsed.

Reply:
That might be an artifact of the github diff -- we're not removing any  of the existing xml fields, we're only adding new fields. There is one change to the meaning of one of the fields: `DateTimeUTC` used to mean different things; sometimes it was `createAt` other times it was `updateAt`/`deleteAt` depending on the post. Now it's always `createAt`. I'll be adding that to the doc notes and blog post.

Reply:
`PostTime` field is gone as far as I can tell :thinking: . That's actually the line the comment referred to.

Reply:
It's now the `CreateAt` field, which exports to the same  `xml:"DateTimeUTC"` field.

Reply:
Oh, thanks, it's serialized to the same, totally missed that. Also it's in the milliseconds now right? A note sounds good, thanks :+1: